includes/wikia/MemcacheSync.class.php

```diff
@@ -24,6 +27,12 @@ function __construct( $cache, $key ) {
 		$this->instance = uniqid ('', true);
 	}
 
+	protected function getLoggerContext() {
+		return [
+			'key' => $this->key,
+		];
+	}
+
 	function lock($time = 5) {
 		if ($this->memc->add($this->lockKey, $this->instance, $time)) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this "if-then-else" statement by a single "return" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1126)